autoload/lsp.vim

```diff
@@ -703,3 +704,31 @@ endfunction
 function! lsp#complete(...) abort
     return call('lsp#omni#complete', a:000)
 endfunction
+
+let s:didchange_queue = []
+let s:didchange_timer = -1
+
+function! s:add_didchange_queue(buf) abort
+    if index(s:didchange_queue, a:buf) != -1
+        return
+	endif
+    call add(s:didchange_queue, a:buf)
+    call lsp#log('s:send_didchange_queue() will be triggered')
+    if exists('*timer_start')
```

Comment:
We don't need to check for `timer_start`, vim-lsp requires lambda and timers to work properly.

Or did you mean `s:didchange_timer` and `s:didchange_timer > 0`